ocs_ci/utility/utils.py

```diff
@@ -2653,6 +2654,26 @@ def convert_device_size(unformatted_size, units_to_covert_to):
     return conversion[units_to_covert_to][units]
 
 
+def convert_bytes_into_right_measure(size_bytes):
```

Comment:
isn't it similar to ocs_ci.utility.utils.convert_device_size?

Reply:
In convert_device_size you have to decide the right measure unit like GI MI TI etc. In my function the measure unit is self calculate for the size. 
My gets only bytes which is what ceph gives in json and convert_device_size is getting any kind of measure. 
convert_bytes_into_right_measure is predicting the measure unit by the size of the input. Similar but different. 
If you don't know what units the outcome will be (GB TB PB etc) it's better to use mine.

Reply:
ok